packages/form-js-editor/src/features/properties-panel/entries/LabelEntry.js

```diff
@@ -51,13 +51,29 @@ export default function LabelEntry(props) {
       editField,
       field,
       isEdited: isFeelEntryEdited,
-      isDefaultVisible: (field) => INPUTS.includes(field.type) || field.type === 'button' || field.type === 'group'
+      isDefaultVisible: (field) => [ ...INPUTS, 'button', 'group', 'table' ].includes(field.type)
     }
   );
 
   return entries;
 }
 
+
+/**
+ * @param {string} type
+ * @returns {string}
+ */
+function getEntryLabel(type) {
```

Comment:
I added exactly that for the `iFrame`, so you probably need to rebase the branch.